src/hooks/useFriction.ts

```diff
@@ -28,26 +30,29 @@ export const useFriction = <M extends HTMLElement | SVGElement = any>({
 
   const { controller } = React.useMemo(() => {
     const interpolators = getInterpolatorsForPairs({ from, to });
+    const maxPosition = getMaxDistanceFriction({
+      mu: config.mu,
+      initialVelocity: config.initialVelocity,
+    });
```

Comment:
We should see a slight performance boost moving this calculation out of the frame loop 😅